cli/src/commands/env.ts

```diff
@@ -0,0 +1,107 @@
+import Command from '../utils/command';
+import flags from '../utils/cli-flags';
+import { getEnvOrDie } from '../utils/helpers';
+import fromPairs from 'lodash.frompairs';
+
+function nonEmpty<T>(x: T[]) { return x.length > 0; }
+
+export default class Env extends Command {
+  public static description = 'manipulate environment of deployed app';
+
+  public static examples = [
+    `$ ${Command.cliBinName} env --list --name fluffy-teapot-76`,
+    `$ ${Command.cliBinName} env --get PGUSER --get PGPASSWORD`,
+    `$ PGPASSWORD=secret ${Command.cliBinName} env --set PGUSER=admin --set-from-env PGPASSWORD`,
+  ];
+
```

Comment:
consider using a hidden command here please, at least until the interface solidify